src/lib.rs

```diff
@@ -105,7 +109,11 @@ impl<N, E, Ix> Dag<N, E, Ix> where Ix: IndexType {
 
     /// Create a new `Dag` with estimated capacity for its node and edge Vecs.
     pub fn with_capacity(nodes: usize, edges: usize) -> Self {
-        Dag { graph: PetGraph::with_capacity(nodes, edges) }
+        let graph = PetGraph::with_capacity(nodes, edges);
+        Dag {
+            graph: graph,
+            cycle_state: DfsSpace::default(),
```

Comment:
Oh, the extra local variable graph is a remnant of an earlier version. I'll amend and fix that.